tests/README.md

```diff
@@ -9,6 +9,22 @@ they're testing).
 
 ## Running the tests
 
+It is possible to run many tests using loopbacked devices to simulate real
+devices. It is also possible to run these tests using real devices specified
+in a configuration file. Both sorts of tests require root permissions and make
+changes to your storage configuration. It is recommended that you run these
+tests only on a dedicated test machine. Running these tests requires that the
+user can sudo.
+
+To run the loopbacked devices tests:
+> make test-loop
```

Comment:
No, test-loop includes the sudo part, to make Travis happy and therefore so does test-real.